### PR TITLE
Added carbon copy feature; Created new route

### PR DESCRIPTION
**Pull request by itsashis4u**
- Added `mashcamp_email.js` with `urlencoded` support as well as `json` support.
- Regex validation of recipient email

### CODE REVIEW

File: mashcamp_email.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+app.use(bodyParser.json());
+
+app.all('*', function(req, res, next) {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Headers', 'X-Requested-With, Authorization, Content-Type');
+    next();
+});
+
+app.listen(4000);
```
Comment (kartikjain) [inline mashcamp_email.js]: if you use the same port number as the earlier node  how would that node run?

File: mashcamp_email.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+console.log('Listening on 4000');
+
+// create reusable transporter object using SMTP transport
+var transporter = nodemailer.createTransport({
+    service: 'gmail',
```
Comment (kartikjain) [inline mashcamp_email.js]: Please change it to zoho.
We are using zoho only..

File: mashcamp_email.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
```
Comment (kartikjain) [inline mashcamp_email.js]: Please take name of the recipient from the front end

File: mashcamp_email.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+app.post('/redspot', function(req, res) {
```
Comment (kartikjain) [inline mashcamp_email.js]: remove redspot end point from this server

**Comment by itsashis4u** [comment]
I have prepared the code as you said. Also tested it on gmail.

File: mashcamp_email.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
..." style="padding: 0;vertical-align: top;text-align: left"> <div><div class="column-top" style="font-size: 32px;line-height: 32px">&nbsp;</div></div><table class="contents" style="border-collapse: collapse;border-spacing: 0;table-layout: fixed;width: 100%"> <tbody><tr> <td class="padded" style="padding: 0;vertical-align: top;padding-left: 32px;padding-right: 32px;word-break: break-word;word-wrap: break-word"> <h3 style="Margin-top: 0;color: #555;font-weight: normal;font-size: 16px;color:#0000CD;line-height: 26px;Margin-bottom: 16px;font-family: Georgia,serif;text-align: center"><strong style="font-weight: bold">About Organisers: MASH Project and Human Circle&#8203;</strong></h3><em><p style="Margin-top: 0;color: #565656;font-family: sans-serif;font-size: 14px;line-height: 24px;Margin-bottom: 24px"><strong style="font-weight: bold">MASH Project</strong>&nbsp;was initiated with a vision to build a community of technology enabled social change makers. Harnessing modern technology, organising various events such as Hackathons and interactive team-building sessions is a decent way to achieve this vision.</p><p style="Margin-top: 0;color: #565656;font-family: sans-serif;font-size: 14px;li
```
Comment (kartikjain) [inline mashcamp_email.js]: email needs to be corrected..

Comment (kartikjain) [inline mashcamp_email.js]: subject needs to be updated

Comment (kartikjain) [inline mashcamp_email.js]: good work here

Comment (itsashis4u) [inline mashcamp_email.js]: I didn't understand. Can you explain?

File: mashcamp_email.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+    if (!/(.+)@(.+){2,}\.(.+){2,}/.test(email)) {
+        res.json({
+            "reply": "Email not valid"
+        });
+        return console.error("Email validation failed");
+    }
+    var mailOptions = {
+        from: 'MASH Project <redspot@mashglobal.org>', // sender address
```
Comment (kartikjain) [inline mashcamp_email.js]: @itsashis4u here we need to change the id which i mentioned above

File: mashcamp_email.js [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+        to: email, // list of receivers
+        cc: ccEmail, // Comma separated list or an array of recipients e-mail addresses
+        subject: 'Thank you for your response ', // Subject line
```
Comment (kartikjain) [inline mashcamp_email.js]: @itsashis4u subject needs to change here

**Pull request merged**